src/gui/clips/AutomationClipView.cpp

```diff
@@ -486,5 +486,57 @@ void AutomationClipView::scaleTimemapToFit( float oldMin, float oldMax )
 	m_clip->generateTangents();
 }
 
+//! Split this Clip.
+/*! \param pos the position of the split, relative to the start of the clip */
+bool AutomationClipView::splitClip( const TimePos pos )
+{
+	setMarkerEnabled( false );
+
+	const TimePos splitPos = m_initialClipPos + pos;
+
+	//Don't split if we slid off the Clip or if we're on the clip's start/end
+	//Cutting at exactly the start/end position would create a zero length
+	//clip (bad), and a clip the same length as the original one (pointless).
+	if ( splitPos > m_initialClipPos && splitPos < m_initialClipEnd )
+	{
+		m_clip->getTrack()->addJournalCheckPoint();
+		m_clip->getTrack()->saveJournallingState(false);
+
+		auto rightClip = new AutomationClip(*m_clip);
+		auto leftClip = new AutomationClip(*m_clip);
+		rightClip->clear();
+		leftClip->clear();
+
+		for( AutomationClip::timeMap::const_iterator it =
```

Comment:
```suggestion
		for(auto it =
```
And bring the next line to this one